addons/website/static/src/components/navbar/navbar.js

```diff
@@ -0,0 +1,123 @@
+/** @odoo-module **/
+
+import { NavBar } from '@web/webclient/navbar/navbar';
+import { useService, useBus } from '@web/core/utils/hooks';
+import { registry } from "@web/core/registry";
+import { patch } from 'web.utils';
+import { EditMenuDialog } from '@website/components/dialog/edit_menu';
+import { OptimizeSEODialog } from '@website/components/dialog/seo';
+import {WebsiteAceEditor, AceEditorAdapterComponent} from '@website/components/ace_editor/ace_editor';
+import {PagePropertiesDialogWrapper} from '@website/components/dialog/page_properties';
+
+const websiteSystrayRegistry = registry.category('website_systray');
+const { useState, onWillStart } = owl;
+
+patch(NavBar.prototype, 'website_navbar', {
```

Comment:
Just in case, let me challenge the specs here. Can't those "fake menu entries" be in another systray item?

Reply:
There were lots of discussion (with FP) regarding the menu organization. We already did not manage to convince to have a dedicated menu for contextual options... so doing a dedicated menu and adding it in the systray, I don't think it will be acceptable :grimacing: